### DISCUSSION TITLE
FlightControl Cmdlets Missing?

### DISCUSSION THREAD

**Discussion by zisk** [CrowdStrike/psfalcon#103] [Q&A] [answered]
Hello

I see documentation for the FlightControl endpoints (https://github.com/CrowdStrike/psfalcon/wiki/Flight-Control) but they don't seem to actually work at the moment even when importing from the master branch. Are they just waiting to be merged in?

**Comment by bk-cs** [comment, marked as answer]
These new commands will be in the next release. I added the commands to the wiki as I was testing them in development.
